Untitled-Engine/Src/Untitled/Core/Log.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2020 [Untitled Games]
```

Comment:
Maybe move this file into its own directory in Core (Untitled-Engine/Src/Untitled/Core/Logger/Log.cpp) same goes for the header.

Reply:
I think that the core directory will not have too many files anyway so didn't want to seperate it out more than is necessary. Do you think we will be expanding the logging functionality to include more than just these 2 files in the future?

Reply:
It is possible we would want to separate out debug and error logging, but it doesn't seem necessary. In either case we could update this in the future if we do decide to expand it.

Reply:
I agree. If we need to extend in the future, we can separate it out more